definitions/subworkflows/pvacseq.wdl

```diff
@@ -78,7 +102,9 @@ workflow pvacseq {
     bam=rnaseq_bam,
     bam_bai=rnaseq_bam_bai,
     min_base_quality=readcount_minimum_base_quality,
-    min_mapping_quality=readcount_minimum_mapping_quality
+    min_mapping_quality=readcount_minimum_mapping_quality,
+    indel_counting_bam=tumorRnaLeftAlignIndels.left_align_indels_bam,
+    indel_counting_bai=tumorRnaLeftAlignIndels.left_align_indels_bam_bai
```

Comment:
Is it desirable or necessary to output this left-aligned bam in the results, to facilitate manual review?

Reply:
Yeah, good idea. Initially I thought no because it was so huge.  But now that it is focused to the regions of interest it seems totally reasonable.  Will make that change.